name: CD2
on:
  workflow_run:
    workflows: [CI]
    types: 
      - completed
      
jobs:
  execute:
    if:  ${{ github.event.workflow_run.conclusion == 'failure' || github.event.workflow_run.conclusion == 'cancelled' }}
    runs-on: ubuntu-latest
    steps: 
      - name: get_job_steps
        uses: actions/github-script@v6
        with:
          script: |
            let workflowRunJobs = await github.rest.actions.listJobsForWorkflowRunAttempt({
              owner: context.repo.owner,
              repo: context.repo.repo,
              run_id: context.payload.workflow_run.id,
              attempt_number: context.payload.workflow_run.run_attempt
            })
            console.log(workflowRunJobs.data.jobs);
            console.log(workflowRunJobs.data.jobs[0].steps);
#            core.setOutput('stepsOutput', stepsOutput);
            
            
#       - name: find_failed_step
#         runs: |
#           ${{ steps.get_job_steps.outputs.stepsOutput
        
        
#       - name: resolve_error
